docs/docs/project-structure.md

```diff
@@ -168,7 +166,9 @@ Routes are how you turn user inputs into Actions. Edit your `src/routes.js` file
 
 Actions are where you define the behavior of your bot. You can add an Action by creating a new `.js` file inside `src/actions`. View **[Actions](/docs/concepts/actions)** and **[Components](/docs/components)** for more details.
 
-<img src="https://botonic-doc-static.netlify.app/images/general/understanding-project.png"/>
```

Comment:
we have this domain to read the static images, why are you changing it to https://i.ibb.co ?